__tests__/app.test.js

```diff
@@ -13,6 +14,21 @@ beforeEach(() => {
 })
 
 describe('app', () => {
+    describe('GET /api', () => {
+        test('GET 200: returns an object containing all the endpoints available', () => {
+            return request(app).get('/api').expect(200).then(({body}) => {
+                const endpointKeys = Object.keys(endpoints)
+                expect(Object.keys(body).length).toBe(endpointKeys.length);
+
+                endpointKeys.forEach((eachEndpointKey) => {
+                    const returnedData = body[eachEndpointKey]
+                    expect(returnedData).toHaveProperty('description')
+                    expect(returnedData).toHaveProperty('queries')
+                    expect(returnedData).toHaveProperty('exampleResponse')
```

Comment:
This is good, you can alternatively just assert that the response matches the endpoints object that was required in with a `toEqual` as that will cover all the bases